### PR TITLE
The focus page

### PR DESCRIPTION
**Pull request by athar05**
- Added routes for focus page 
- Added localStorage 
- Created Context to manage state 
- Made API calls for weather

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Name from "./pages/Name";
 import { Images } from "../src/backend/images";
 import { Routes, Route } from "react-router-dom";
+import Focus from "./pages/Focus";
 
 function App() {
   const randNum = Math.floor(Math.random() * Math.floor(Images.length));
```
Comment (bhavyaj12) [inline src/App.js:7]: Remove console logs in final code

File: src/index.css [hunk 2]
```diff
@@ -75,3 +82,57 @@ align-items: center;
...
+.welcome-message {
+  padding: 0.5rem 0.5rem;
+  color: white;
+  background-color: black;
+  opacity: 0.8;
+  border-radius: 0.25rem;
+}
+
+.quote {
+  background-color: black;
```
Comment (bhavyaj12) [inline src/index.css:95]: You should create a variable for a color used so often

File: src/index.css [hunk 3]
```diff
@@ -75,3 +82,57 @@ align-items: center;
...
+.quote-component {
+  min-width: 100vw;
+  text-align: center;
+}
+
+.focus-parent {
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+  /* justify-content: space-around; */
```
Comment (bhavyaj12) [inline src/index.css:136]: Remove commented css

**Review by bhavyaj12 — COMMENTED** [review]
Logic is easy to understand. Good breakdown of the various components. Good to go

**Review by teja538 — APPROVED** [review]
code looks good and clean
just remove the commented code and use variables for colors
everything else is great

**Pull request merged**